pkg/plugin/state.go

```diff
@@ -327,7 +326,7 @@ func (r *nodeOtherResourceState) calculateReserved(memSlotSize *resource.Quantit
 
 // totalReservableCPU returns the amount of node CPU that may be allocated to VM pods -- i.e.,
 // excluding the CPU pre-reserved for system tasks.
-func (s *nodeState) totalReservableCPU() uint16 {
+func (s *nodeState) totalReservableCPU() api.MilliCPU {
```

Comment:
note: this is called in `pkg/plugin/plugin.go`, used as: `totalCpu := ...`. That variable should be renamed to `totalMilliCpu`. (there's also `maxTotalCpu` below it, which should be renamed in the same way).